app/javascript/packages/device/index.js

```diff
@@ -1,11 +1,24 @@
+/**
```

Comment:
Can we add / update specs for this file.

https://github.com/18F/identity-idp/blob/main/spec/javascripts/packages/device/index-spec.js